### PR TITLE
branch commit

### PR DESCRIPTION
**Pull request by javagurulv**

### CODE REVIEW

File: teacher/lesson_4_git_brancing_pull_request_code_review/material/Lesson_plan.txt [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
-1. Team disputes, how to solve it, ideal solution not exist,
-   team work always is a compromise, sacrifice yourself for the team
-2. Stand up meeting (agile practice)
-3. Git branches
+1. Team disputes, how to solve it,
+   ideal solution not exist,
+   team work always is a compromise,
+   sacrifice yourself for the team.
+
+2. Stand up meeting (agile practice).
```
Comment (AndrejsB86) [inline teacher/lesson_4_git_brancing_pull_request_code_review/material/Lesson_plan.txt:8]: some comment

Comment (Darxney) [reply teacher/lesson_4_git_brancing_pull_request_code_review/material/Lesson_plan.txt:8]: yeah nah

File: teacher/lesson_4_git_brancing_pull_request_code_review/material/Lesson_plan.txt [hunk 2]
```diff
@@ -1,9 +1,14 @@
 Lesson 4 plan:
 
-1. Team disputes, how to solve it, ideal solution not exist,
-   team work always is a compromise, sacrifice yourself for the team
-2. Stand up meeting (agile practice)
-3. Git branches
+1. Team disputes, how to solve it,
```
Comment (Alexandr-Kozhekin) [inline teacher/lesson_4_git_brancing_pull_request_code_review/material/Lesson_plan.txt:3]: mu comit

File: teacher/lesson_4_git_brancing_pull_request_code_review/material/Lesson_plan.txt [hunk 3]
```diff
@@ -1,9 +1,14 @@
 Lesson 4 plan:
 
-1. Team disputes, how to solve it, ideal solution not exist,
-   team work always is a compromise, sacrifice yourself for the team
-2. Stand up meeting (agile practice)
```
Comment (AlexaMaksimovic) [inline teacher/lesson_4_git_brancing_pull_request_code_review/material/Lesson_plan.txt:5]: comment

File: teacher/lesson_4_git_brancing_pull_request_code_review/material/Lesson_plan.txt (hunk 1 above)
Comment (javagurulv) [reply teacher/lesson_4_git_brancing_pull_request_code_review/material/Lesson_plan.txt:8]: answer

**Review by Darxney — APPROVED** [review]
i approve this

**Review by NikitaBobrovs — APPROVED** [review]
Approved

**Pull request merged**